android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/scenario/AuthorizedAccount.java

```diff
@@ -21,6 +21,9 @@ public class AuthorizedAccount {
     @Nullable
     public final String accountLabel;
     @Nullable
+    public final Uri accountIcon;
+    @Deprecated
```

Comment:
Can we add a deprecation description here, so that consumers know to use `accountIcon` instead?

Reply:
AFAIK, there is no way to add a deprecation description with the @Deprecated annotation used here. the only fields it accepts are `since` and `forRemoval`. I could add a JavaDoc message instead?

Reply:
I added a JavaDoc comment with the deprecation message. this notifies consumers to use `accountIcon` instead in the IDE tooltip